build.gradle

```diff
@@ -357,3 +364,23 @@ project(':java-sdk-demo-build') {
 		}
 	}
 }
+
+spotless {
+	java {
+		// licenseHeaderFile	rootProject.file('spotless-java-sdk-demo/spotless.license.java')			// License header file
```

Comment:
if we don't use this license header file, can we just delete it (and also remove this out-commented line)?

Reply:
Deleted.